app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -746,4 +855,58 @@ public void shareMessage(String message) {
             currContext.startActivity(sendIntent);
         }
     }
-}
\ No newline at end of file
+
+
+    public class getWebSearch extends AsyncTask<Void, Void, Void> {
```

Comment:
Why are you making network calls using asynctask?
We are already using retrofit, so use it.